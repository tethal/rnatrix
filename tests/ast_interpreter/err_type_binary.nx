fun main() {
    5 + null;
}
