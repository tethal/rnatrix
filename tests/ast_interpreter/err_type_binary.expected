<string>:2:7: error: operator + cannot be applied to Int and Null
    5 + null;
      ^
